text/0037-minting.md

```diff
@@ -0,0 +1,162 @@
+- Feature Name: `minting`
+- Start Date: 2022-04-21
+- MCIP PR: [mobilecoinfoundation/mcips#0037](https://github.com/mobilecoinfoundation/mcips/pull/37)
+- Tracking Issue: N/A
+
+# Summary
+[summary]: #summary
+
+A normal MobileCoin transfer transaction contains a set of inputs, which are marked spent, to create (mint) a new set of outputs. The network validators confirm that the sum of the value of the inputs is equal to the sum of the value of the outputs, using Ring Confidential Transactions.
+
+To support multiple assets, some of which represent backed assets, we will use an auditable process to introduce new token types to the MobileCoin blockchain, following a minting and burning procedure that corresponds with the status of the backed assets.
+
+The birds-eye view is that after a set of _Governors_ has been agreed upon by the MobileCoin consensus validator node operators and signed by the MobileCoin Foundation, a new _Minting Configuration_  is submitted to specify the signers who are allowed to mint the new asset type on the MobileCoin blockchain. An amount of the backing asset will be verifiably locked, for example into a smart contract on another chain, and then a _Minting Transaction_ for the locked amount establishes a new set of transaction outputs for which there were no inputs marked spent on the MobileCoin blockchain, using the _confidential token ID_ (see [MCIP #25](https://github.com/mobilecoinfoundation/mcips/pull/25)) for the corresponding backing asset. When the backing funds are desired to be released from the lock, the amount to be released will be _verifiably burned_ (see [MCIP #35](https://github.com/mobilecoinfoundation/mcips/pull/35)) on the MobileCoin blockchain.
```

Comment:
This is actually a guide-level explanation and not a summary, IMO. We should move this and the rest of the text in this section to the guide-level explanation section.